minimal_high_density.md

```diff
@@ -0,0 +1,103 @@
+# `minimal_high_density.slim`: a minimal model suitable for high-density simulations
+
+**code:** [minimal_high_density.slim](minimal_high_density.slim); on [github](https://github.com/kr-colab/spatial_sims_standard/blob/main/minimal_high_density.slim)
+
+This code implements the same model as [`minimal.slim`](minimal.html),
+but implements some "map-based" alternative computational methods so that runtime is linear in density
+instead of quadratic.
+However, these methods are somewhat approximate and may lead to discretization artifacts if density is low,
+so if neighborhood sizes are small (below, say 5 or 10), then the methods should probably not be used.
+
+There are two aspects of the "standard" methods demonstrated in `minimal.slim` that are quadratic in population density
+(i.e., quadratic in the parameter `K`): local density computation, and mate choice.
+The two methods can be used independently - for instance, maybe your simulation
+has small interaction neighborhood size but large mating neighborhood size,
+so you'd use these methods for mating but not density computations.
```

Comment:
A little vague what "these methods" refers to.  Maybe it would be helpful to actually give the method shown in this example a name, so you can refer to it?  Otherwise, "the alternative method shown here" instead of "these methods", I guess.